rpc_jobs/standard_job_checkmarx.yml

```diff
@@ -0,0 +1,57 @@
+- job-template:
+    CRON: "{CRON_WEEKLY}"
+    RPC_GATING_BRANCH: master
+    NUM_TO_KEEP: 30
+    READ_ONLY_TEST: true
+    SLAVE_TYPE: "shared"
+    name: '{trigger}-Checkmarx_{scan_type}-{scan_repo_name}'
+    project-type: pipeline
+    pipeline-scm:
+      scm:
+        - git:
+            url: https://github.com/rcbops/rpc-gating
+            branches:
+              - "${{RPC_GATING_BRANCH}}"
+            credentials-id: "github_account_rpc_jenkins_svc"
+      script-path: job_dsl/standard_job_checkmarx.groovy
+    concurrent: true
+    properties:
+      - build-discarder:
+          num-to-keep: "{NUM_TO_KEEP}"
+      - github:
+          url: "{scan_repo_url}"
+      - inject:
+          properties-content: |
+            BOOT_TIMEOUT=900
```

Comment:
Does this have any use if it's being run on a shared slave?

Reply:
Heh probably not, I'll remove that.

Reply:
Done